emsdk latest-fastcomp: error: use of undeclared identifier 'input_buffer'

**Godot version:**
master


**OS/device including version:**
Arch Linux


**Issue description:**
```Swift
[user@archlinux godot]$ scons p=javascript target=release tools=no -j 8
scons: Reading SConscript files ...

scons: warning: EnsureSConsVersion is ignored for development version
File "/run/media/user/user/git/godot/SConstruct", line 3, in <module>
WebM SIMD optimizations are disabled. Check if your CPU architecture, CPU bits or platform are supported!
Checking for C header file mntent.h... (cached) yes
scons: done reading SConscript files.
scons: Building targets ...
[  1%] Compiling ==> platform/javascript/audio_driver_javascript.cpp
[  8%] platform/javascript/audio_driver_javascript.cpp:66:2: error: use of undeclared identifier 'input_buffer_write'; did you mean 'capture_buffer_write'?
        input_buffer_write(sample32);
        ^~~~~~~~~~~~~~~~~~
        capture_buffer_write
./servers/audio_server.h:63:7: note: 'capture_buffer_write' declared here
        void capture_buffer_write(int32_t sample);
             ^
platform/javascript/audio_driver_javascript.cpp:201:2: error: use of undeclared identifier 'input_buffer_init'; did you mean 'capture_buffer_init'?
        input_buffer_init(buffer_length);
        ^~~~~~~~~~~~~~~~~
        capture_buffer_init
./servers/audio_server.h:62:7: note: 'capture_buffer_init' declared here
        void capture_buffer_init(int driver_buffer_frames);
             ^
platform/javascript/audio_driver_javascript.cpp:248:2: error: use of undeclared identifier 'input_buffer'
        input_buffer.clear();
        ^
3 errors generated.
shared:ERROR: '/run/media/user/user/git/emsdk/fastcomp/fastcomp/bin/clang++ -target asmjs-unknown-emscripten -D__EMSCRIPTEN_major__=1 -D__EMSCRIPTEN_minor__=38 -D__EMSCRIPTEN_tiny__=41 -D_LIBCPP_ABI_VERSION=2 -Werror=implicit-function-declaration -Xclang -nostdsysteminc -Xclang -isystem/run/media/user/user/git/emsdk/fastcomp/emscripten/system/include/libcxx -Xclang -isystem/run/media/user/user/git/emsdk/fastcomp/emscripten/system/lib/libcxxabi/include -Xclang -isystem/run/media/user/user/git/emsdk/fastcomp/emscripten/system/include/compat -Xclang -isystem/run/media/user/user/git/emsdk/fastcomp/emscripten/system/include -Xclang -isystem/run/media/user/user/git/emsdk/fastcomp/emscripten/system/include/libc -Xclang -isystem/run/media/user/user/git/emsdk/fastcomp/emscripten/system/lib/libc/musl/arch/emscripten -Xclang -isystem/run/media/user/user/git/emsdk/fastcomp/emscripten/system/local/include -DEMSCRIPTEN -c -std=c++11 -Os -fno-exceptions -fno-rtti -Wall -Werror=return-type -DJAVASCRIPT_ENABLED -DUNIX_ENABLED -DNO_THREADS -DNO_SAFE_CAST -DJAVASCRIPT_EVAL_ENABLED -DNDEBUG -DPTRCALL_ENABLED -DGDSCRIPT_ENABLED -DMINIZIP_ENABLED -DZSTD_STATIC_LINKING_ONLY -DFREETYPE_ENABLED -DSVG_ENABLED -DHAVE_MNTENT -Ithirdparty/nanosvg -Ithirdparty/freetype/include -Ithirdparty/libpng -Ithirdparty/zstd -Ithirdparty/zlib -Iplatform/javascript -I. -Ieditor -mllvm -disable-llvm-optzns platform/javascript/audio_driver_javascript.cpp -fno-inline-functions -Xclang -disable-O0-optnone -Xclang -isystem/run/media/user/user/git/emsdk/fastcomp/emscripten/system/include/SDL -c -o platform/javascript/audio_driver_javascript.javascript.opt.bc -emit-llvm' failed (1)
[ 12%] Compiling ==> thirdparty/bullet/BulletCollision/CollisionShapes/btConvexHullShape.cpp
[ 12%] Compiling ==> thirdparty/bullet/BulletCollision/CollisionShapes/btConvexInternalShape.cpp
[ 12%] Compiling ==> thirdparty/bullet/BulletCollision/CollisionShapes/btConvexPointCloudShape.cpp
[ 12%] Compiling ==> thirdparty/bullet/BulletCollision/CollisionShapes/btConvexPolyhedron.cpp
[ 12%] Compiling ==> thirdparty/bullet/BulletCollision/CollisionShapes/btConvexShape.cpp
[ 13%] Compiling ==> thirdparty/bullet/BulletCollision/CollisionShapes/btConvex2dShape.cpp
[ 13%] scons: *** [platform/javascript/audio_driver_javascript.javascript.opt.bc] Error 1
Compiling ==> thirdparty/bullet/BulletCollision/CollisionShapes/btConvexTriangleMeshShape.cpp
scons: building terminated because of errors.
[user@archlinux godot]$ 

```


**Steps to reproduce:**
```
emsdk install latest-fastcomp
emsdk activate latest-fastcomp
source emsdk_env.sh
scons p=javascript target=release tools=no -j 8
```

--- a/platform/javascript/audio_driver_javascript.cpp
+++ b/platform/javascript/audio_driver_javascript.cpp
@@ -63,7 +63,7 @@ void AudioDriverJavaScript::mix_to_js() {
 void AudioDriverJavaScript::process_capture(float sample) {
 
 	int32_t sample32 = int32_t(sample * 32768.f) * (1U << 16);
-	input_buffer_write(sample32);
+	capture_buffer_write(sample32);
 }
 
 Error AudioDriverJavaScript::init() {
@@ -198,7 +198,7 @@ void AudioDriverJavaScript::finish() {
 
 Error AudioDriverJavaScript::capture_start() {
 
-	input_buffer_init(buffer_length);
+	capture_buffer_init(buffer_length);
 
 	/* clang-format off */
 	EM_ASM({
@@ -244,8 +244,6 @@ Error AudioDriverJavaScript::capture_stop() {
 
 	});
 	/* clang-format on */
-
-	input_buffer.clear();
 
 	return OK;
 }

